lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -550,10 +550,17 @@ static std::optional<Value> createToLayoutOp(PatternRewriter &rewriter,
 
   auto desiredLayout = rewriter.getAttr<LayoutAttr>(
       ty, desiredMemorySpace, currLayout.getGrid(), desiredElementType);
+  tensor::EmptyOp exising_empty = input.getDefiningOp<tensor::EmptyOp>();
+
+  // If we are changing the layout of existing op
+  // we need to set the insertion point to the existing op
+  if (exising_empty) {
+    rewriter.setInsertionPoint(input.getDefiningOp());
+  }
+
```

Comment:
I think it's safe, and potentially more correct, to just unconditionally `setInsertionPoint`:
```c++
rewriter.setInsertionPoint(input.getDefiningOp());
auto output = rewriter.create<tensor::EmptyOp>(....
```

Reply:
I think problem is when we want to layout function argument, getDefiningOp returns nullptr